integrations/spark/build.gradle

```diff
@@ -164,8 +152,8 @@ shadowJar {
 
     manifest {
         attributes(
-                "Premain-Class": 'marquez.spark.agent.MarquezAgent',
-                "Agent-Class": 'marquez.spark.agent.MarquezAgent',
+                "Premain-Class": 'openlineage.spark.agent.SparkAgent',
```

Comment:
Since this jar will still be published by Marquez, we'll want the keep `marquez.*`

Reply:
we can't change that, cause we don't have agent in marquez, we have only listener. This is an entry for javaagent, so it will try to find such class. By Mike comment, I've removed that SparkAgent from marquez code.

Reply:
Ahh, makes sense 👍